plugins/importexport/onix30/filter/MonographONIX30XmlFilter.inc.php

```diff
@@ -248,13 +248,14 @@ function createProductNode($doc, $submission, $publicationFormat) {
 		/* --- License information --- */
 
 		$publication = $submission->getCurrentPublication();
+		$pubLocale = $publication->getData('locale');
 
 		if ($publication->isCCLicense()) {
 			AppLocale::requireComponents(LOCALE_COMPONENT_PKP_SUBMISSION); // For CC License Names
 			$licenseOpts = Application::getCCLicenseOptions();
 			$licenseUrl = $publication->getData('licenseUrl');
 			if (array_key_exists($licenseUrl, $licenseOpts)) {
-				$licenseName = (__($licenseOpts[$licenseUrl], [], $publication->getData('locale')));
+				$licenseName = (__($licenseOpts[$licenseUrl], [], $pubLocale));
```

Comment:
Running into an issue here where the license name isn't being retrieved when the UI is set to french (didn't test other languages), even though the en_US locale is being passed.

Reply:
You would need to use:
`PKPLocale::requireComponents(LOCALE_COMPONENT_PKP_SUBMISSION, $publication->getData('locale'));`
before getting the license name...